internal/dast/dast_local_scan.go

```diff
@@ -56,10 +54,21 @@ func DASTLocalScan(httpClient *http.Client, nullifyHost string, input *DASTLocal
 	}
 	defer client.Close()
 
+	imageRef := "public.ecr.aws/nullify/dast:0.1.0"
```

Comment:
is this the part that we still need to do?

Reply:
Yes, we need to upload the image to ghcr or ecr